docs/source/changelog.rst

```diff
@@ -2,11 +2,6 @@
 Changelog
 #########
 
-****************
-Upcoming release
-****************
```

Comment:
I sorta like this section, but it's not really used unfortunately. It'd also only make sense for this to be used during latest so I'm fine with this being removed